<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CsvImportDialogClass</class>
 <widget class="QDialog" name="CsvImportDialogClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">CSV Import</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/pic/spread.png</normaloff>:/pic/spread.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string notr="true"> Datei </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="fileLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filePushButton">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="icon">
           <iconset resource="res/res.qrc">
            <normaloff>:/pic/folder_blue_open.png</normaloff>:/pic/folder_blue_open.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string notr="true">Trennungszeichen: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSep">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <property name="minimumContentsLength">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string notr="true">;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">,</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string notr="true">Textkodierung: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxCodec"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <italic>false</italic>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string notr="true">Erntejahr:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="displayFormat">
           <string notr="true">yyyy</string>
          </property>
          <property name="date">
           <date>
            <year>2009</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLoad">
          <property name="toolTip">
           <string notr="true">CSV Datei laden</string>
          </property>
          <property name="text">
           <string notr="true">Laden</string>
          </property>
          <property name="icon">
           <iconset resource="res/res.qrc">
            <normaloff>:/pic/reload.png</normaloff>:/pic/reload.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string notr="true"> Betrieb </string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string notr="true">Betriebsname:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditUserName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditUserName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string notr="true"> Zuordnung </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxGemark">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string notr="true">Gemarkung</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxGemark">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="checkBoxFieldNr">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string notr="true">Schlag-Nr</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="comboBoxFieldNr">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBoxFlur">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string notr="true">Flur</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxFlur">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="checkBoxFieldName">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string notr="true">Schlagname</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="comboBoxFieldName">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBoxLotNr">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string notr="true">Flurstücks-Nr</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxLotNr">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="checkBoxFieldLotSize">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string notr="true">Nutzfläche ohne LE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="comboBoxFieldLotSize">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBoxLotSize">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string notr="true">Katasterfläche</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxLotSize">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QCheckBox" name="checkBoxCropCode">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string notr="true">Code der Kultur</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QComboBox" name="comboBoxCropCodeNr">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboBoxLotSubNr">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBoxLotSubNr">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string notr="true">Unter-Nr</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonHelp">
       <property name="text">
        <string notr="true">Hilfe</string>
       </property>
       <property name="icon">
        <iconset resource="res/res.qrc">
         <normaloff>:/pic/question.png</normaloff>:/pic/question.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="toolTip">
        <string notr="true">Daten importieren</string>
       </property>
       <property name="text">
        <string notr="true">Importieren</string>
       </property>
       <property name="icon">
        <iconset resource="res/res.qrc">
         <normaloff>:/pic/down.png</normaloff>:/pic/down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="toolTip">
        <string notr="true">Import abbrechen</string>
       </property>
       <property name="text">
        <string notr="true">Abbrechen</string>
       </property>
       <property name="icon">
        <iconset resource="res/res.qrc">
         <normaloff>:/pic/remove.png</normaloff>:/pic/remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBoxGemark</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxGemark</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLotNr</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxLotNr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLotSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxLotSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFieldNr</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxFieldNr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFieldName</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxFieldName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFieldLotSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxFieldLotSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCropCode</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxCropCodeNr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CsvImportDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>656</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFlur</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxFlur</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLotSubNr</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxLotSubNr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
